### PR TITLE
doc: add package.conf example, closes HYB-558

### PR DESCRIPTION
**Pull request by Obero**

### CODE REVIEW

**Comment by Obero** [comment]
Draft until feature release

**Comment by Obero** [comment]
Please discuss common subjects about `package.conf` example here: https://github.com/gatling/gatling-sbt-plugin-demo/pull/31

File: .gatling/example.package.conf [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Learn more about Configuration as Code : https://docs.gatling.io/reference/execute/cloud/user/configuration-as-code
+
```
Comment (bastien-gatling) [inline .gatling/example.package.conf:4]: I would add a new comment line here, like : `All configurations are optional`

**Comment by Isammoc** [comment]
Removed myself as reviewer from this one, because only comments will be done in [sbt's PR](https://github.com/gatling/gatling-sbt-plugin-demo/pull/31)

**Review by tpetillot — APPROVED** [review]

**Comment by Obero** [comment]
> Removed myself as reviewer from this one, because only comments will be done in [sbt's PR](https://github.com/gatling/gatling-sbt-plugin-demo/pull/31)

It's ready, I bring you back :D

**Review by Isammoc — APPROVED** [review]

**Review by bastien-gatling — APPROVED** [review]

**Pull request merged**